optuna/integration/allennlp/_pruner.py

```diff
@@ -53,9 +51,8 @@ def wrapper(subclass: Any, *args: Any, **kwargs: Any) -> Any:
 
 
 def _create_pruner(
-    pruner_class: Optional[str],
-    pruner_keys: Optional[str],
-    pruner_values: Optional[str],
+    pruner_class: str,
+    pruner_params: Dict[str, Any],
```

Comment:
`pruner_params` is used as kwargs like `pruner(**pruner_params)`, so I think `pruner_kwargs` may show the usage better.
I should mention it in the previous PR, though.

Reply:
It makes sense, thank you for the comment. Updated in c09f27f.

Reply:
Thanks!